conformity/resource_conformity_profile_test.go

```diff
@@ -21,61 +22,104 @@ func TestAccResourceconformityProfile(t *testing.T) {
 				Config: testAccCheckConformityProfileBasic(ttl),
 				Check: resource.ComposeTestCheckFunc(
 					resource.TestCheckResourceAttr("conformity_profile.rtm002", "name", "test-with-rules"),
-					resource.TestCheckResourceAttr("conformity_profile.rtm002", "included.0.id", "RTM-002"),
-					resource.TestCheckResourceAttr("conformity_profile.rtm002", "included.0.exceptions.0.tags.0", "some_tag"),
-					resource.TestCheckResourceAttr("conformity_profile.rtm002", "included.0.extra_settings.0.value", ttl),
-					resource.TestCheckResourceAttr("conformity_profile.rtm002", "included.1.extra_settings.0.values.0.value", "includeConformityOrganization"),
-				), ExpectNonEmptyPlan: true,
-			},
-			{
-				Config: testAccCheckConformityProfileBasic(ttlUpdate),
-				Check: resource.ComposeTestCheckFunc(
-					resource.TestCheckResourceAttr("conformity_profile.rtm002", "included.0.extra_settings.0.value", ttlUpdate),
+					resource.TestCheckResourceAttr("conformity_profile.rtm002", "included.0.id", "RG-001"),
+
+					resource.TestCheckResourceAttr("conformity_profile.rtm002", "included.1.id", "RTM-002"),
+					resource.TestCheckResourceAttr("conformity_profile.rtm002", "included.1.exceptions.0.tags.0", "some_tag"),
+					resource.TestCheckResourceAttr("conformity_profile.rtm002", "included.1.extra_settings.0.value", ttl),
+
+					resource.TestCheckResourceAttr("conformity_profile.rtm002", "included.2.id", "SNS-002"),
+					resource.TestCheckResourceAttr("conformity_profile.rtm002", "included.2.extra_settings.0.values.0.value", "includeConformityOrganization"),
 				), ExpectNonEmptyPlan: true,
 			},
+			// {
+			// 	Config: testAccCheckConformityProfileBasic(ttlUpdate),
+			// 	Check: resource.ComposeTestCheckFunc(
+			// 		resource.TestCheckResourceAttr("conformity_profile.rtm002", "included.1.extra_settings.0.value", ttlUpdate),
+			// 	), ExpectNonEmptyPlan: true,
+			// },
```

Comment:
❓

Reply:
Thank you. Will uncomment these lines. This is only for testing.